epilogue-processor/src/test/java/edu/wpi/first/epilogue/processor/AnnotationProcessorTest.java

```diff
@@ -59,6 +59,50 @@ public void update(EpilogueBackend backend, Example object) {
     assertLoggerGenerates(source, expectedGeneratedSource);
   }
 
+  @Test
+  void implicitOptInLogging() {
+    String source =
+        """
+      package edu.wpi.first.epilogue;
+
+      class Example {
+        @Logged double x;
+        @Logged int y;
+
+        @Logged public double getValue() { return 2.0; }
+        @Logged public boolean isActive() { return false; }
```

Comment:
These would be better in a separate test, so we can check that _just_ a logged field is enough to make a class logged, and that _just_ a logged method is also enough. Having both in one test means it's ambiguous, and also means that if either field or method detection breaks (but not both), then it wouldn't be caught by the test cases.

```java
@Test
void implicitLoggedFields() {
  String source =
    """
    package edu.wpi.first.epilogue;

    class Example {
      @Logged double x;
    }
    """;

  // ...
}

@Test
void implicitLoggedMethods() {
  String source =
    """
    package edu.wpi.first.epilogue;

    class Example {
      @Logged
      public double x() { return 42; }
    }
    """;

  // ...
}
```

Additionally, it'd be good to have another test to cover the null case of a class with _no_ `@Logged` annotation and with _no_ implicitly logged members to make sure we don't generate loggers for things that users didn't opt into